### DISCUSSION TITLE
Hiding text using server when splash screen loaded

### DISCUSSION THREAD

**Discussion by baonguyen91hnx** [ionic-team/capacitor#3783] [Q&A] [answered]
Hello guys.
Im developing a new app by using Capacitor, everything work fine and now when we build app for production, when the app loading splash screen , it show a small text below the screen with Using app server http://example.com, it pretty annoy and show a domain where my app pointed to.
Im using server.url in  my capacitor.config.json :

`  
  "server": {
    "url": "http://example.com"
  }
`
Anyone have a ideal to hide this text when app loading or config to disable it ? 
Thank you

**Comment by jcesarmobile** [comment, marked as answer]
server url is not meant to be used in production apps, only for development for pointing to live reload servers.

The toast with the url should only appear when you run a debug build (when you run from Android Studio/Xcode), if you build and install a release build then it won't appear. So not sure what you mean by "build for production", but doesn't look like you are running a release build.

**Reply by baonguyen91hnx** [reply]
Sorry for not make a clear question right from the beginning, im using capacitor on existing vuejs project so i load all it through webview, that why i need to use server url, because my project is quite huge and i want to reuse them.
Sorry for my bad english

**Reply by ubersensei** [reply]
Is it correct to conclude that "server url" can be used in production apps and that the toast will disappear in production ?
